packages/repository/README.md

```diff
@@ -238,62 +31,50 @@ The repository module provides APIs to define LoopBack 3.x data sources and
 models. For example,
 
 ```ts
-import {
-  DataSourceConstructor,
-  juggler,
-  Entity,
-  model,
-  ModelDefinition
-} from '@loopback/repository';
+// src/datasources/db.datasource.ts
+import {juggler, DataSourceConstructor} from '@loopback/repository';
 
-export const ds: juggler.DataSource = new DataSourceConstructor({
+export const db: juggler.DataSource = new DataSourceConstructor({
   name: 'db',
   connector: 'memory',
 });
+```
+
+```ts
+// src/models/note.model.ts
+import {model, Entity, property} from '@loopback/repository';
 
 @model()
 export class Note extends Entity {
-  static definition = new ModelDefinition({
-    name: 'note',
-    properties: {
-      id: {name: 'id', type: 'number', id: true},
-      title: 'string',
-      content: 'string'
-    }
-  })
-};
+  @property({id: true})
+  id: string;
+  @property() title: string;
+  @property() content: string;
+}
 ```
 
 **NOTE**: There is no declarative support for data source and model yet in
-LoopBack Next. These constructs need to be created programmatically as
+LoopBack 4. These constructs need to be created programmatically as
 illustrated above.
 
 ### Define a repository
```

Comment:
Make it capitalization for titles. 
--> Define a Repository

Reply:
confirmed with @bschrammIBM , your original version was correct (for capitalization) but should be `Defining a repository`. 
Similarly for others.. e.g. `Defining a controller`.  
That's something she can edit later.